cypress/tests/resource-provider-card.spec.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  PVC_NAME,
+  DATA_SOURCE_NAME_NSFS,
+  DATA_SOURCE_NAME_AWS,
+} from '../constants/tests';
+import { dataSourceAWS, dataSourceNSFS } from '../mocks/data-source';
+import { MCGMSCommon } from '../views/mcg-ms-common';
+
+describe('admin dashboard - resource provider card', () => {
+  before(() => {
+    cy.login();
+  });
+
+  after(() => {
+    cy.logout();
+  });
+
+  beforeEach(() => {
+    MCGMSCommon.visitMcgMsDashboard();
```

Comment:
use [wait](https://github.com/avanthakkar/mcg-ms-console/blob/8499327164665c01e81b7a5896d438f471eb2e1b/cypress/views/common.ts#L5) to wait till we have all the skeleton resolved